index.md

```diff
@@ -3,15 +3,55 @@ layout: lesson
 root: .  # Is the only page that doesn't follow the pattern /:path/index.html
 permalink: index.html  # Is the only page that doesn't follow the pattern /:path/index.html
 ---
-FIXME: home page introduction
 
-<!-- this is an html comment -->
+This tutorial is built to teach you ESMValTool.  
 
-{% comment %} This is a comment in Liquid {% endcomment %}
+> ## What will you learn in this course
+>
+>   - What is ESMValTool
+>
+>   - How to install ESMValTool
+>
+>   - How to run ESMValTool
+>
+>   - How to develop your own diagnostics and recipes
+>
+>   - How to contribute your recipes and diagnostics back into ESMValTool
+>
+{: .checklist}
 
 > ## Prerequisites
 >
-> FIXME
+> Basic understanding of git (optional - but useful)
+> Basic understanding of your preferred command line interface (ie a bash terminal)
+> Access to CMIP data
+> Access to a suitable computing system (eg Jasmin)
+> Github account (optional, but useful!)
```

Comment:
Single line breaks do not render in markdown. I would suggest to make a list or two separate lists, with optional and minimal requirements. Does the suitable computing system belong to the optional or minimal list? I would say you own laptop would also work for simple and small recipes and especially this tutorial should be doable on a home computer. 
```suggestion
> *Minimal requirements:*
>   - Basic understanding of your preferred command line interface (i.e. a bash terminal)
>   - Access to CMIP data
>   - Access to a suitable computing system (eg Jasmin)
>
> *Optional, but useful:*
>   - Basic understanding of git
>   - Github account
```